CMakeLists.txt

```diff
@@ -571,13 +588,14 @@ if(BUILD_TESTS)
       NAME reconfiguration_test
       PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/reconfiguration.py
       CONSENSUS raft
+      ADDITIONAL_ARGS --raft-election-timeout 4000
```

Comment:
Lowering the raft election timeout here (default is 100s) as the reconfiguration test now also waits for a new election to complete.